python/cli.schema.json

```diff
@@ -582,6 +583,21 @@
         "status",
         "message"
       ]
+    },
+    "SearchResponse": {
+      "title": "SearchResponse",
+      "type": "object",
+      "properties": {
+        "search_results": {
+          "type": "array",
+          "items": {
+            "type": "object"
```

Comment:
@shauryr can you make this a typed reference?
Also, you should run the `yarn codegen` to regenerate the `src/api/types.ts` definitions for typescript.